Fuzzlyn/Execution/ProgramExecutor.cs

```diff
@@ -73,19 +73,20 @@ ProgramResult RunAndGetResult(byte[] bytes)
         // Launches a new instance of Fuzzlyn to run the specified programs in.
         public static List<ProgramPairResults> RunSeparately(List<ProgramPair> programs)
         {
-            string dotnet;
+            string host;
             using (Process proc = Process.GetCurrentProcess())
             using (ProcessModule mm = proc.MainModule)
             {
-                dotnet = mm.FileName;
+                host = mm.FileName;
             }
 
             string fuzzlyn = Assembly.GetExecutingAssembly().Location;
             string fuzzlynDir = Path.GetDirectoryName(fuzzlyn);
+            bool hostIsFuzzlyn = host.Equals("fuzzlyn", StringComparison.OrdinalIgnoreCase);
```

Comment:
This should be comparing `fuzzlyn`, not the literal string `"fuzzlyn"`. Also it should not be ignoring case since Linux file systems are case sensitive. So just make it `bool hostIsFuzzlyn = fuzzlyn == host;`

Reply:
Fixed. This way it should interactively matrix open-source value while working on both Windows and Linux.